src/transformers/models/mbart/modeling_mbart.py

```diff
@@ -842,6 +956,7 @@ def __init__(self, config: MBartConfig, embed_tokens: Optional[nn.Embedding] = N
         )
         self.layers = nn.ModuleList([MBartEncoderLayer(config) for _ in range(config.encoder_layers)])
         self._use_flash_attention_2 = config._attn_implementation == "flash_attention_2"
+        self._use_sdpa = config._attn_implementation == "sdpa"
```

Comment:
this is something we want to avoid, let's just test `config._attn_implementation` directly

Reply:
I'm not sure I understand what you are saying when you say "test config._attn_implementation directly"

Reply:
I was copying this: https://github.com/huggingface/transformers/blob/62c60a30181a65e1a3a7f19c3055a240a6a21335/src/transformers/models/bart/modeling_bart.py#L982

Reply:
cc our offline conv!